gcloud-java-core/pom.xml

```diff
@@ -90,4 +90,102 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-site-plugin</artifactId>
+        <version>3.4</version>
+        <configuration>
+          <skipDeploy>true</skipDeploy>
+          <reportPlugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-changelog-plugin</artifactId>
+              <version>2.3</version>
+            </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-project-info-reports-plugin</artifactId>
+              <version>2.8</version>
+              <reportSets>
+                <reportSet>
+                  <reports>
+                    <report>index</report>
+                    <report>dependency-info</report>
+                    <report>dependencies</report>
+                    <report>dependency-convergence</report>
+                    <report>project-team</report>
+                    <report>mailing-list</report>
+                    <report>cim</report>
+                    <report>issue-tracking</report>
+                    <report>license</report>
+                    <report>scm</report>
+                    <report>dependency-management</report>
+                    <report>distribution-management</report>
+                    <report>summary</report>
+                    <report>modules</report>
+                  </reports>
+                </reportSet>
+              </reportSets>
+              <configuration>
```

Comment:
If I understand your proposal correctly, that's effectively what Marco tried in the [second commit](https://github.com/mziccard/gcloud-java/commit/7afcc34c56b19556ef2b2cc8bcbc7459604910ad), but that led to the project reports being dropped.  The default merging strategy (described in the second link from your post above) should have overridden just the artifactId, but for some reason other configurations were dropped.